packages/wallets/src/types.ts

```diff
@@ -39,8 +39,10 @@ export namespace THooks {
 export namespace TPlatforms {
     export type All = MT5 | OtherAccounts | SortedMT5Accounts;
     export type MT5 = THooks.AvailableMT5Accounts['platform'];
-    export type OtherAccounts =
-        | Parameters<NonNullable<ReturnType<typeof useCreateOtherCFDAccount>['mutate']>>[0]['payload']['platform'];
+    export type OtherAccounts = Exclude<
+        Parameters<NonNullable<ReturnType<typeof useCreateOtherCFDAccount>['mutate']>>[0]['payload']['platform'],
+        'derivez'
```

Comment:
As DerivEZ is no longer available, updated the types to exclude checks for DerivEZ